test/services-test.js

```diff
@@ -0,0 +1,558 @@
+'use strict';
+
+const testUrl = 'http://localhost:3000';
+
```

Comment:
These tests are increaible
Great job mocking all of your data.
but each serverice should have its own test file,
this helps keep code more maintainable